routes/api/contacts.js

```diff
@@ -1,25 +1,57 @@
-const express = require('express')
+const express = require("express");
+const {
+  getContacts,
+  getContactById,
+  removeContact,
+  addContact,
+  checkRequiredParams,
+  updateContact,
+} = require("../../models/contacts");
+const bodyParser = require("body-parser");
 
-const router = express.Router()
+const router = express.Router();
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.get("/", async (req, res, next) => {
+  const contacts = await getContacts();
+  res.json({ data: JSON.parse(contacts) });
+});
 
-router.get('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.get("/:contactId", async (req, res, next) => {
+  const contact = await getContactById(req.params.contactId);
+  res.json(contact ? { data: contact } : { message: "Not found" });
```

Comment:
wrong. You will return different message but what about HTTP status? Should it be always the same?